packages/devtools_app/test/support/cli_test_driver.dart

```diff
@@ -76,7 +76,7 @@ class CliAppFixture extends AppFixture {
 
   static Future<CliAppFixture> create(String appScriptPath) async {
     final Process process = await Process.start(
-      Platform.resolvedExecutable,
+      Platform.isWindows ? 'dart.bat' : 'dart',
```

Comment:
This doesn't seem right - Dart is a .exe on Windows (only Flutter and Pub are bat's)?

Reply:
Definitely seems to fail on the bots too. I'll open a PR to revert this bit and send you the review on the assumption it's a mistake.

```
 ProcessException: The system cannot find the file specified.
  
    Command: dart.bat --observe=0 --pause-isolates-on-start test/fixtures/logging_app.dart
```